daemon/indexing/listener/listener.js

```diff
@@ -1,9 +1,9 @@
 const fs = require('fs')
 const http = require('http')
 const urllib = require('url')
+const Origin = require('origin')
```

Comment:
While this is right in the long term, this change would break things right now, since the NPM origin.js isn't up to date. Will NPM link work here? Where does it need to happen? In the container?

Reply:
Yea that is right. I'll use `npm link` to link it to a version of `origin-js` that I build. You can see an [example here](https://github.com/OriginProtocol/origin-box/blob/master/docker-compose.yml#L167) for `origin-dapp`. It lets me put the container setups in place without worrying about maintaining that relative import and I don't have to build `origin-js` in each of the containers for the event listener and apollo server.

Reply:
Should it be in a separate container like dapp is, and wait for `origin-js` to build the same way dapp does it?

Reply:
Yep exactly, that is what I'm going to do. You can see my initial pass [here](https://github.com/OriginProtocol/origin-box/commit/28514da3476ff2839dfcabe3d082864a645003d0).